src/Amazon.Extensions.S3.Encryption.csproj

```diff
@@ -41,8 +41,8 @@
     </PropertyGroup>
 
     <ItemGroup>
```

Comment:
Can you bump the version of the library?

Reply:
Done. Thanks for catching it.